content/docs/concepts/architecture.md

```diff
@@ -0,0 +1,320 @@
+    - source_selector:
+      routes:
+      - destination_selector:
+          service: envoy-proxy
+```
+
+If a {{<color "#0050EF" >}}Client{{< /color >}} provided no 'source labels' with its Request for the 'service-mesh' Network Service, it would not match the 'service: envoy-proxy'
+for the first match, and so would fall through to the final 'catch all' match with no source_selector, and be matched to {{<color "#008A00" >}}Endpoint{{< /color >}}
+that advertised a Network Service named 'service-mesh' with 'destination label' 'service: envoy-proxy'.
+
+If a {{<color "#0050EF" >}}Client{{< /color >}} provided a 'source label' of 'service: envoy-proxy' it would match the first match and be matched to an {{<color "#008A00" >}}Endpoint{{< /color >}}
+that advertised a Network Service named 'service-mesh' with 'destination label' 'service: vl3'
+
+### Registry Domains
+
+Network Service Mesh allows multiple independent mutually ignorant Registry Domains.
+
+The Network Service Registry Domain of a Network Service is indicated by suffixing an '@domain' to the Network Service Name.
+
+So for example, a Network Service named 'service-mesh' in the 'finance.example.com' domain would be 'service-mesh@finance.example.com'
+
+The reference implementation of Network Service Mesh locates the Registry Server for a Registry Domain by looking up a SRV record
+for the name of the domain.  This is not the only permissible way to do it, it is done as one example that permits scaling to
+'internet scale'.
+
+## Inter-domain
+
+A {{<color "#0050EF" >}}Client{{< /color >}} may request a Network Service from any Network Service Registry Domain independent of where it is running.
+Whether the lookup from the Registry for that Registry Domain is permitted, and whether the {{<color "#0050EF" >}}Client{{< /color >}} is permitted to connect
+to that Network Service is a matter of policy, not a matter of where the {{<color "#0050EF" >}}Client{{< /color >}} is running.
+
+An {{<color "#008A00" >}}Endpoint{{< /color >}} may register as providing a Network Service in any Network Service Registry Domain independent of where it is running.
+Whether the {{<color "#008A00" >}}Endpoint{{< /color >}} is permitted to register in that Registry Domain is a matter of policy, not a matter of where the {{<color "#008A00" >}}Endpoint{{< /color >}} is running.
+
+### Floating Inter-domain
+
+A Network Service Registry Domain need not be associated directly with *any* Runtime Domain.  It may be a purely logical
+Registry, with {{<color "#0050EF" >}}Clients{{< /color >}} and {{<color "#008A00" >}}Endpoints{{< /color >}} running across many different Runtime Domains that the Registry Domain has no direct association with.
+
+When run in this mode, it is referred to as a 'floating' Registry Domain.
+
+## Advanced Features
+
+Network Service Mesh's 'match' process for selecting candidate {{<color "#008A00" >}}Endpoints{{< /color >}} to provide a Network Service can be used to
```

Comment:
I'd just put a list here with a links of the features below it.

Reply:
List provided as suggested.